### ISSUE TITLE
Question: generate CHANGELOG.md for existing git repo

### ISSUE THREAD

**Issue by bethmal** [semantic-release/changelog#150]
Hi,

We have an existing gitlab repo. which contains many tags created previously. How can I configure semantic-release/changelog to create CHANGLOG.md for all of those previously created tags, not only for the latest version bumped by semantic-release?

Thanks.

**Comment by Nosmoht** [comment]
I have the same use case. Is it possible somehow?

**Comment by trallnag** [comment]
Yes, it is possible! I figured it out after a bunch of trial and error and finding the correct hint in the documentation. I documented it partially in one of my repositories: <https://github.com/trallnag/testbench-semantic-release#regenerate-changelog>
